spec/examples/teacher_training_api/course_list_response.json

```diff
@@ -17,6 +17,7 @@
         ],
         "changed_at": "2019-06-13T10:44:31Z",
         "code": "3GTY",
+        "uuid": "903c6f3c-b2d6-46e1-8bf7-3fbd13d3ea09",
```

Comment:
ah actually the course code is equal on both those fixtures so best keep the uuid the same

Reply:
Done =)